gen/eduroam_block.pl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+
+local $::SERVICE_NAME = "eduroam_block";
+local $::PROTOCOL_VERSION = "3.0.0";
```

Comment:
We can keep protocol version 3.0.0, it just has to match same property in slave script ``eduroam_block.sh``, which does.